lib/engine/game/g_1822/game.rb

```diff
@@ -1387,11 +1388,8 @@ def bidbox_start_private
         end
 
         def can_gain_extra_train?(entity, train)
-          if train.name == self.class::EXTRA_TRAIN_PULLMAN
-            return false if entity.trains.any? { |t| t.name == self.class::EXTRA_TRAIN_PULLMAN }
-          elsif self.class::EXTRA_TRAIN_PERMANENTS.include?(train.name)
-            return false if entity.trains.any? { |t| self.class::EXTRA_TRAIN_PERMANENTS.include?(t.name) }
-          end
+          return false if pullman_train?(train) && entity.trains.any? { |t| pullman_train?(t) }
```

Comment:
Should this be disallowed or only forced discard?

Reply:
Rules say "may never own more than one pullman", so I think you can acquire the private, but then have to discard.